src/rgw/rgw_rest_sts.cc

```diff
@@ -191,6 +196,8 @@ RGWOp *RGWHandler_REST_STS::op_post()
      }
   }
 
+  free(data);
```

Comment:
I think I don't like this "api"--we shouldn't have to call free on a char pointer in code written in 2018